packages/cli/lib/LocalDevManager.js

```diff
@@ -47,6 +46,7 @@ class LocalDevManager {
     this.isGithubLinked = options.isGithubLinked;
     this.watcher = null;
     this.uploadWarnings = {};
+    this.runnableComponents = this.getRunnableComponents(options.components);
```

Comment:
We now find project components outside the `LocalDevManager` so we can determine what apps a project contains and therefore what account to recommend the user use. Components are passed into the constructor, so I moved the logic to find runnable components from setup to the constructor